flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/strategy/SchedulingTopology.java

```diff
@@ -43,11 +43,33 @@
 	 */
 	Optional<SchedulingExecutionVertex> getVertex(ExecutionVertexID executionVertexId);
 
+	/**
+	 * Looks up the {@link SchedulingExecutionVertex} for the given {@link ExecutionVertexID}.
+	 *
+	 * @param executionVertexId identifying the respective scheduling vertex
+	 * @return The respective scheduling vertex or throw {@link Exception} if the vertex does not exist
```

Comment:
* It is misleading to write that we potentially throw `Exception` because `IllegalArgumentException` is unchecked. 
* You can use @throws tag

```
* @return The respective scheduling vertex
* @throws IllegalArgumentException If the vertex does not exist
```

Reply:
OK